rfcs/text/0001-wildcard-data-type.md

```diff
@@ -432,28 +372,28 @@ Stage 2: Document new concerns or resolutions to previously listed concerns. It'
 
 ### Wildcard and case-insensitivity
 
-Some fields require flexibility in how users search. Their content is messy (e.g. user-agent) or popular for threat hunters (e.g. file paths and names, command line processes), and a single character in the opposite casing can bypass a detection today for `keyword` fields. The `wildcard` field provides improved performance of leading `wildcard` and `regex` term-level queries, but is also a step towards case-insensitive search support in Elasticsearch. As Elasticsearch moves forward towards introducing a case-insensitive query option [3], ECS considers the fields adopting `wildcard` to be popular candidates for case-insensitive searching once the feature is available.
+Some fields require flexibility in how users search. For example, their content is messy (such as user-agent values) or popular for threat hunters (command line processes), and a single character in the opposite casing can bypass a detection today for `keyword` fields. The `wildcard` field provides improved performance of leading `wildcard` and `regex` term-level queries but is also a step towards case-insensitive search support in Elasticsearch.
```

Comment:
Both types support case insensitive search, since 7.10 IIRC. Might want to revisit this sentence

```suggestion
Some fields require flexibility in how users search. For example, their content is messy (such as user-agent values) or popular for threat hunters (process command-lines or a PowerShell script). The `wildcard` field provides improved performance of leading `wildcard` and `regex` term-level queries.
```

Reply:
We use EQL for all of our security rules that require explicit case insensitivity, because it makes it easy to go back and forth

Reply:
Thanks - will revisit!